tryoutmanager/models.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3.6
+
+import os
+import uuid
+import sqlalchemy as sa
+
+from datetime import datetime
+from sqlalchemy import Column, String, Boolean, DateTime, Table, \
+                       ForeignKey, Enum, Index, UniqueConstraint
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.declarative import declarative_base
+
+db = sa.create_engine(os.environ['DATABASE_URL'])
+metadata = sa.MetaData()
+
+Base = declarative_base(metadata)
+Session = sessionmaker(bind=db)
+
+
+# Association tables for many-to-many relationships
+markers_challenges = Table('markers_challenges',
+                           Base.metadata,
```

Comment:
I'd prefer you use 1 indent in the continuation for this case since it isn't a function def. 

this would look like : 
```
markers_challenges = Table(
    'markers_challenges',
    Base.metadata,
    Column('marker_email',String,ForeignKey('markers.email')),
    Column('challenge_id',UUID,ForeignKey('challenges.id')))
```
And yes, this is also pep8

Reply:
Both approaches are PEP8 according to flake8. Why the preference for this one?

Reply:
I find this easier to read because this is declaring a table. I also didn't like that due to how far it was indented, it required each column and its parameters to be spread across many lines. Notice I put a line break after each comma.

Reply:
PEP8 [doesn't actually list that style as valid](https://www.python.org/dev/peps/pep-0008/#indentation); the closest thing would be multiline indentation with no formal params on the first line, no vertical alignment with the opening paren, an extra indent (vs. your commented snippet), and multiple formal params per line.

Reply:
It doesn't explicitly say that you can't reserve one line for one parameter. It specifies what not to do. 

Here is another opinion on the matter but at the end of the day, I want it like that. It's a lot closer to how I write other languages and I find it cleaner. 
[someone on StackOverflow](https://stackoverflow.com/questions/6609956/python-function-argument-list-formatting)